test/controllers/DraftMetadataChecksResultsControllerSpec.scala

```diff
@@ -34,22 +38,6 @@ class DraftMetadataChecksResultsControllerSpec extends FrontEndTestHelper {
   }
 
   "DraftMetadataChecksResultsControllerSpec GET" should {
```

Comment:
I know you didn't write this but can you please change "DraftMetadataChecksResultsControllerSpec GET" to `DraftMetadataChecksResultsController GET`?